runtime/vm/LayoutFFITypeHelpers.hpp

```diff
@@ -116,6 +116,10 @@ class LayoutFFITypeHelpers
 			typeCode = J9NtcPointer;
 			break;
 		case FFI_TYPE_STRUCT:
+#if defined(J9ZOS390)
+		case FFI_TYPE_STRUCT_FF:
```

Comment:
Please add comments for the `case` conditions. e.g.
```
case FFI_TYPE_STRUCT:      /* Fall through */
case FFI_TYPE_STRUCT_FF:   /* Fall through */
```

Reply:
Changed in https://github.com/eclipse-openj9/openj9/pull/19645/commits/1e752555416853814f4beb0cf0eea5f3fe8580bf

Reply:
> Please add comments for the `case` conditions. e.g.
> 
> ```
> case FFI_TYPE_STRUCT:      /* Fall through */
> case FFI_TYPE_STRUCT_FF:   /* Fall through */
> ```

@ChengJin01 Comments are only appropriate if there's some code between one case label and the next.